docs/dawson/usecase.md

```diff
@@ -1,89 +1,323 @@
+* Please cite [Dawson et al. (2021)](https://doi.org/10.17603/ds2-68a9-0s64) if using any of the Texas FEMA Hurricane data used in the examples below. 
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
 
-### Description 
+### ADCIRC
+
+For more information on running ADCIRC and documentation, see the following links:
+
+* [ADCIRC Wiki](https://wiki.adcirc.org/wiki/Main_Page)
+* [ADCIRC Web Page](https://adcirc.org/)
+
+ADCIRC is available as a standalone app accesible via the [DesignSafe front-end](https://www.designsafe-ci.org/rw/workspace/#!/ADCIRC::Simulation).
+
+ADCIRC files used in this demo are pre-staged on TACC resources that DesignSafe execution systems have access to, at the path `/work/06307/clos21/pub/adcirc`. See the section on using data from Projects directory for using other data sources.
+
+### Tapis
+
+Tapis is the main API to control and access HPC resources with. For more resources and tutorials on how to use Tapis, see the following:
+
+- [Tapis CLI](https://tapis-cli.readthedocs.io/en/latest/contents.html)
+* [AgavePy](https://tacc-cloud.readthedocs.io/projects/agave/en/latest/)
+* [DesignSafe Webinar](https://www.youtube.com/watch?v=-_1lNWW8CAg&t=1854s&ab_channel=NHERIDesignSafe-CIMedia)
+
+To initialize tapis in our jupyter notebook we use AgavePy. Relies on `tapis auth init --interactive` being run from CLI first.
+
+```python
+from agavepy.agave import Agave
+
+ag = Agave.restore()
+```
+
+### Pylauncher
+
+Pylauncher is a parametric job launcher used for launching a collection of HPC jobs within one HPC job. By specifying a list of jobs to execute in either a CSV or json file, pylauncher manages resources on a given HPC job to execute all the jobs using the given nodes. Inputs for pylauncher look something like (for csv files, per line):
+
+```
+num_processes,<pre process command>;<main parallel command>;<post process command>
+```
+
+The pre-process and post-process commands are executed in serial, while the main command is executed in parallel using the appropriate number of processes. Note pre and post process commands should do light file management and movement and no computationally intensive tasks.
+
+## Tapis Pylauncher App
+
+The pylauncher app is the main application we will be using to run ensemble simulations. It serves as a wrapper around the [TACC pylauncher](https://github.com/TACC/pylauncher) utility. 
+
+Check out the application from the github page - https://github.com/UT-CHG/tapis-pylauncher and deploy it using either the Tapis CLI or AgavePy (See documentation links above under Tapis section), or email cdelcastillo21@gmail.com for access to an already deployed version of the application (it is not a public application yet, so has to be shared explicitly with users).
```

Comment:
```suggestion
Check out the application from the github page - [https://github.com/UT-CHG/tapis-pylauncher](https://github.com/UT-CHG/tapis-pylauncher) and deploy it using either the Tapis CLI or AgavePy (See documentation links above under Tapis section), or email cdelcastillo21@gmail.com for access to an already deployed version of the application (it is not a public application yet, so has to be shared explicitly with users).
```

Reply:
We should discuss the best way to provide access to the App.

Reply:
Yeah let me know what you think. Easiest way would be to make it a public app. Because for a user to deploy the app themselves they have to open a ticket to get app deployment privileges, and its a mess.